client/src/components/Editor.jsx

```diff
@@ -161,6 +186,7 @@ export default function Editor({schema, endpoint, setQuery}) {
 
     /* Execute Current Operation in Query Pane (cmd + enter OR auto) */
   const execOperation = async function () {
+    console.log('HERELOL');
```

Comment:
lol ya sure you still want this log here?

Reply:
absolutely